src/client/d3/newUserIntro.js

```diff
@@ -0,0 +1,180 @@
+const { svg } = require('./setup.js');
+
+function newUserIntro() {
+  const falseDataArray = [{ heading: 'test', id: 100, index: 0, likes: 5, visits: 1, x: 215, y: 170 }];
+
+  const rScale = d3
+.scaleSqrt()
+.domain([0, d3.max(falseDataArray, d => d.likes)])
+.range([3, 8]);
+
+  function zoomed() {
+    d3.select('.memory-group').attr('transform', d3.event.transform);
+  }
+
+  const fdGrp = svg
+  .append('g')
+  .attr('class', 'memory-group')
+  .call(d3.zoom()
+    .scaleExtent([1 / 3, 3])
+    .on('zoom', zoomed));
+  const nodeGrp = fdGrp
+      .append('g')
+        .attr('class', 'nodes');
+
+  const nodes = nodeGrp
+      .selectAll('circle.node')
```

Comment:
There is too much repetition here,

You are adding lots of complexity + branching with this feature.

I there no way you can use the old functionallity. Just pass in you mock data and run the tutorial, You are redefining to much stuff here.

Its not maintainable or good